xchange-stream-service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -459,18 +475,24 @@ private void onIdle(ChannelHandlerContext ctx) {
   }
 
   protected void handleChannelMessage(String channel, T message) {
-    NettyStreamingService<T>.Subscription subscription = channels.get(channel);
-    if (subscription == null) {
-      LOG.debug("Channel has been closed {}.", channel);
-      return;
-    }
-    ObservableEmitter<T> emitter = subscription.emitter;
-    if (emitter == null) {
-      LOG.debug("No subscriber for channel {}.", channel);
-      return;
+    if ("ALL".equals(channel)) {
```

Comment:
ALL seems to be a constant associated only with CoinbasePro should we either pull it out of CoinbasePro and make it a Netty constant or override in the CoinbasePro `handleChannelMessage`

Reply:
yes, this is in the wrong place, It should be specific to coinbase

Reply:
moved to CoinbaseProStreamingService